circle.yml

```diff
@@ -47,7 +50,7 @@ step-library:
   - &save-cache
       save_cache:
         key: 'v3/{{ .Environment.CIRCLE_JOB }}/{{ arch }}/{{ .Branch }}/{{ checksum ".circle-week" }}'
-        paths: [ "node_modules", "/root/.ccache", "mason_packages/.binaries" ]
+        paths: [ "node_modules", "/root/.ccache", "~/.ccache", "mason_packages/.binaries" ]
```

Comment:
I guess we could also export `CCACHE_DIR=/root/.ccache` in the iOS CI jobs instead of adding this.

Reply:
Thanks, I’ll look into that. `/root` may not exist or may have permissions issues on macOS.

Reply:
I tried this and:

- We’d have to export the cache path for every macOS-based job — it’s not enough to do this in a step, it has to be global for the job.
- We’d either have to create the `/root/.ccache` directory with `sudo` and give it proper permissions, or do something similar with a symlink.

I think we should just leave it as-is, including both possible ccache paths. Caches aren’t shared across platforms and a non-existent path doesn’t break the save/restore steps.